releases/tickets/v29_1_0.rst

```diff
@@ -0,0 +1,93 @@
+- :jira:`DM-50218`: HealPixelization: Error computing envelope for point region [sphgeom]
+- :jira:`DM-50243`: Document how to set logging values for commands in bps [ctrl\_bps]
+- :jira:`DM-50294`: Writing to quantumGraphGeneration.out is buffered [ctrl\_bps]
+- :jira:`DM-50310`: Add data ID / dimension record tooling for QGs and zip archives [pipe\_base]
+- :jira:`DM-50379`: Add missing metadata method to ObjectSizeNoSourcesError [meas\_extensions\_psfex]
+- :jira:`DM-50400`: pass IDDS\_MAX\_NAME\_LENGTH to remote build task [ctrl\_bps\_panda]
+- :jira:`DM-50446`: Make it possible for define-visits to fix bad detector-regions even when visit is unchanged [obs\_base]
+- :jira:`DM-50451`: Make sure dataset\_ids and UUID literals work in butler string expressions and the butler CLI [pipe\_base]
+- :jira:`DM-50476`: Fix metadata-existence skipping in auto-retries and add job-level timing info to metadata [ctrl\_mpexec]
+- :jira:`DM-50490`: Record butler metrics in quantum metadata [ctrl\_mpexec, utils]
+- :jira:`DM-50491`: Add get/put timing metrics into butler [pipe\_base]
+- :jira:`DM-50498`: Make bps config environment values strings [ctrl\_bps]
+- :jira:`DM-50506`: Ensure HTTPResourcePath closes connection when doing partial reads [resources]
+- :jira:`DM-50538`: Add more logging to define-visits to track down partial visit\_detector\_region coverage [obs\_base]
+- :jira:`DM-50539`: Improve QG builder error handling and improve adjust\_all\_quanta [pipe\_base]
+- :jira:`DM-50559`: Fix AlgorithmError metadata abc [pipe\_base]
+- :jira:`DM-50614`: Add ability to auto-release every held job when using ctrl\_bps\_htcondor [ctrl\_bps\_htcondor]
+- :jira:`DM-50645`: Connection pool warnings with RSP butler.get at USDF [resources]
+- :jira:`DM-50661`: Don't constrain on raw existence in define-visits CLI script [obs\_base]
+- :jira:`DM-50724`: Support bulk deletes in ResourcePath [resources]
+- :jira:`DM-50727`: Database deadlock errors when attempting to delete dataset from /rubin/embargo [daf\_butler\_admin]
+- :jira:`DM-50753`: Use vcr for cp\_pipe efd tests [cp\_pipe]
+- :jira:`DM-50857`: Investigate performance of butler remove-runs in embargo [resources]
+- :jira:`DM-50885`: Add a --datasetTypes switch to transfer\_from\_graph to allow parallel merges of large quantumGraph [ctrl\_mpexec, pipe\_base]
+- :jira:`DM-50897`: Parallelize s3 ResourcePath.mremove [resources]
+- :jira:`DM-50926`: Change default color cycle in plotting utils [utils]
+- :jira:`DM-50958`: Try to improve database performance removing large collections [daf\_butler\_migrate]
+- :jira:`DM-50969`: Push spatial predicate terms and data ID tables into spatial-overlap subquery when possible [pipe\_base]
+- :jira:`DM-50973`: use a map file to reduce the size of the bps PanDA submission size [ctrl\_bps\_panda]
+- :jira:`DM-51073`: Make middleware v29.1 release [daf\_butler, pipe\_base, resources]
```

Comment:
Probably makes sense to drop this from the release notes, since it's a little self-referential.